bin/UniProt-Protein-Annotation-NF.R

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env Rscript
+
+# Libraries to get annotations using the uniprot web service
+# The docker image on quay.io is out of date, so pull a docker image of
+# bioconductor, and then install
+# Be sure to set the library path to a temporary directory first. 
+dir.create('./Rlibs', showWarnings = FALSE)
+.libPaths('./Rlibs')
+BiocManager::install('UniProt.ws', lib = './Rlibs')
```

Comment:
again I think this needs to already be integrated into the docker image

Reply:
For stability reasons of the workflow and also ensuring the version you are using is what you actually want

Reply:
Yes, agreed! See my above comment about this